Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -1086,19 +1086,24 @@ private void HandlePositionAssigned(OrderEvent fill)
         /// <summary>
         /// Gets the amount of time since the last call to algorithm.Portfolio.ProcessFill(fill)
         /// </summary>
-        private TimeSpan TimeSinceLastFill
+        protected virtual TimeSpan TimeSinceLastFill
         {
-            get { return DateTime.UtcNow - new DateTime(Interlocked.Read(ref _lastFillTimeTicks)); }
+            get { return CurrentTimeUtc - new DateTime(Interlocked.Read(ref _lastFillTimeTicks)); }
         }
 
         /// <summary>
         /// Gets the date of the last sync (New York time zone)
         /// </summary>
-        private DateTime LastSyncDate
+        protected DateTime LastSyncDate
         {
             get { return new DateTime(Interlocked.Read(ref _lastSyncTimeTicks)).ConvertFromUtc(TimeZones.NewYork).Date; }
         }
 
+        /// <summary>
+        /// Gets current time UTC. This is here to facilitate testing
```

Comment:
Thank you for the comment describing _why_ it's here :)

Reply:
might want to add a similar comment via `<remarks></remarks>` to the other two members who are now protected (and one other is virtual) -- presumably for the sole purpose for supporting unit tests.

no action require -- just me commenting --, taking hidden members and making them visible _just_ to support unit testing is usually indicative of a missing component/class/abstraction that would have public members. In this case it would be some 'thing' who's responsibility it is to manage the brokerage cash synchronization process.